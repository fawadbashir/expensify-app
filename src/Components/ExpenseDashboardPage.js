import React from 'react';
import { Link } from 'react-router-dom'
import Button from 'react-bootstrap/Button'


import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'
import ExpensesSummary from './ExpensesSummary'

const ExpenseDashboardPage = () => (
    <div>
        <div className='pt-3 full-width-body' style={{ marginBottom: '0.5rem', padding: '0.5rem 0', }}>
            <ExpensesSummary />
            <Link to='/create'><Button size='lg' className=' mt-3' style={{ fontSize: '18px', backgroundColor: '#3D9970', color: 'white', border: 'none', padding: '10px' }} >Add Expense</Button></Link>
        </div>
        <ExpenseListFilters />
        <ExpenseList />



    </div >
)

export default ExpenseDashboardPage