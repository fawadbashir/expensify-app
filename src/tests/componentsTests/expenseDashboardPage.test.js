import React from 'react'
import { shallow } from 'enzyme'
import {createSerializer} from 'enzyme-to-json';
import  ExpenseDashboardPage  from '../../Components/ExpenseDashboardPage'

expect.addSnapshotSerializer(createSerializer({mode: 'shallow'}));

test('should Render Dashboard Expense Page', () => {
    const wrapper = shallow(<ExpenseDashboardPage/>)
    expect(wrapper).toMatchSnapshot()
})