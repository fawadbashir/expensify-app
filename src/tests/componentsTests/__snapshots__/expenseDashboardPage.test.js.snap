// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should Render Dashboard Expense Page 1`] = `
<div>
  This is my Dashboard Page
  <Connect(expenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
